### PR TITLE
Needed command_runsub glue to work

### PR DESCRIPTION
**Pull request by vague666**

### CODE REVIEW

**Comment by dequis** [comment]
You've included unrelated changes to `clones_scanner.pl`, do this to revert them:

```
git reset HEAD^ scripts/clones_scanner.pl
git commit --amend
```

And edit the commit message there to mention `act.pl` in the commit message.

Commit contents aside, is this something that broke in 0.8.18/0.8.19 (like #268 suggests) or something that broke in the last change to the script?

**Comment by ailin-nemui** [comment]
I think it was actually broken by dgl in that simplification. also @vague666 please update $VERSION so that the fix will be picked up

File: scripts/act.pl [hunk 1]
```diff
@@ -30,9 +30,20 @@
...
+      Irssi::command_runsub('act', $data, $server, $item);
+    }
+    else {
+      _act(1);
+    }
+};
+
 Irssi::command_bind('act public', sub { _act(2); });
 Irssi::command_bind('act all', sub { _act(3); });
+Irssi::command_bind('act', sub { _act(1); });
```
Comment (ailin-nemui) [inline scripts/act.pl]: This line is no longer needed I believe

File: scripts/clones_scanner.pl [hunk 2]
```diff
@@ -79,6 +79,7 @@
 
 
 Irssi::settings_add_time('clones_scanner', 'clones_scanner_maxtime', 900);
+Irssi::settings_add_bool('clones_scanner', 'clones_scanner_include_ident', 0);
```
Comment (ailin-nemui) [inline scripts/clones_scanner.pl]: The clone scanner made it into your pr again

**Pull request closed without merging**